### DISCUSSION TITLE
AttributeError: 'builtin_function_or_method' object has no attribute 'default'

### DISCUSSION THREAD

**Discussion by Candy-Crusher** [pyg-team/pytorch_geometric#9683] [Q&A]
Hello, I'm using pytorch 1.11.0 cuda 11.3 torch-geometric 2.6.1
When I try to import torch_geometric, there is an attributeError. How can I solve this problem?

```
Python 3.8.12 (default, Oct 12 2021, 13:49:34) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch_geometric
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/torch_geometric/__init__.py", line 8, in <module>
    from .index import Index
  File "/opt/conda/lib/python3.8/site-packages/torch_geometric/index.py", line 461, in <module>
    @implements(aten.clone.default)
AttributeError: 'builtin_function_or_method' object has no attribute 'default'
>>> 
```

**Comment by akihironitta** [comment]
I believe it's because of the PyTorch version. I'd suggest trying it with PyTorch 1.13+ as per https://github.com/pyg-team/pytorch_geometric/releases/tag/2.6.0.

**Comment by Imkalus** [comment]
I have the same problem. Have you solved it？

**Reply by gieses** [reply]
Upgrading to 1.13.1 from 1.10 did the job for me.

**Comment by Kiumb1223** [comment]
you might try using **a lower version of torch-geometric** to resolve this issue~   :)
I just encountered the same question . And here is my original environmental setup: `Python == 3.8.1, torch == 1.8.1+cu111, torch-geometric == 2.6.1`
When I attempted to import torch-geometric, I also encountered  an AttributeError.
To fix it ,  I downgraded `torch-geometric` to version like 1.7.2 or 2.0.0 ,which I solve the issue~
